libraries/Microsoft.Bot.Builder.Azure/Microsoft.Bot.Builder.Azure.csproj

```diff
@@ -45,6 +45,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.4.4" />
```

Comment:
>Azure.Storage.Blobs [](start = 31, length = 19)

Hmm....I think we should conder creating a new assembly.  Doing it this way means I have dependencies on both the old and the new Blob sdk.  Ick

Reply:
New assemblies :)